yb-voyager/src/queryissue/queryissue.go

```diff
@@ -36,12 +37,29 @@ func NewParserIssueDetector() *ParserIssueDetector {
 	return &ParserIssueDetector{}
 }
 
-func (p *ParserIssueDetector) GetIssues(query string) ([]issue.IssueInstance, error) {
+func (p *ParserIssueDetector) GetIssues(query string, targetDbVersion *version.YBVersion) ([]issue.IssueInstance, error) {
 	parseTree, err := queryparser.Parse(query)
 	if err != nil {
 		return nil, fmt.Errorf("error parsing query: %w", err)
 	}
-	return p.getDMLIssues(query, parseTree)
+	issues, err := p.getDMLIssues(query, parseTree)
+	if err != nil {
+		return nil, fmt.Errorf("error getting DML issues: %w", err)
+	}
+
+	// Filter out issues that are fixed in the target DB version.
+	var filteredIssues []issue.IssueInstance
```

Comment:
currently you are just filtering these here we probably need this for analyzeschema Issues separately, until we move all of them here

Reply:
Yeah, but until we move all issues from analyze schema to here, there is no way to do it, because the analyze schema issues have no "MinimumFixedInVersion" information 😬 .

Reply:
@priyanshi-yb I realise that the changes to the Issue (analyzeschema) struct to add minversion is in the next PR https://github.com/yugabyte/yb-voyager/pull/1916/files#diff-367fd9bfcb63a06bd262e78a25a905f2932646cd60c86733ef59222888c3b348 . 

I will take this up as part of that?

Reply:
Yes sure

Reply:
Please also mention that in the PR description.